### DISCUSSION TITLE
How to contribute and get involved?

### DISCUSSION THREAD

**Discussion by easybakechef** [ros2-rust/ros2_rust#66] [General]
Hi! I would like to contribute to the project. What's the best way to get started?

**Comment by keenanjohnson** [comment]
I also have this question :)

**Comment by jhdcs** [comment]
I'd say the best way to get involved would be to take a look at our issues list, and see if there is anything that you can work on, or try to tackle.

Also, a lot of discussions about `ros2_rust` happen on the ROS 2 Rust Working Group's Matrix channel (https://matrix.to/#/+rosorg-rust:matrix.org). Feel free to drop by and say hello!

Lastly - try and use the library! While a lot of work is currently focused on adding features to the library, feedback about how it performs and how comfortable it is to use is always helpful! It also helps us know where we need to add more documentation, or more tests/checks! We might even use the feedback to make more issues for more people to try contributing to `ros2_rust`!

**Reply by swarajpeppermint** [reply]
When trying to connect to the Matrix Channel, seeing this message 
![image](https://user-images.githubusercontent.com/85288063/166501712-a715c22a-fae0-413c-870b-c8361e710478.png)

Any idea where the discussions are happening?

**Reply by jhdcs** [reply]
That seems odd. Perhaps try this link?

https://matrix.to/#/!AiYXdTWExqamTdatBl:matrix.org?via=matrix.org&via=matrix.bobedibob.de
